cloudflared/rootfs/etc/s6-overlay/scripts/cloudflared-config.sh

```diff
@@ -14,7 +14,7 @@ checkConfig() {
     bashio::log.trace "${FUNCNAME[0]}"
     bashio::log.info "Checking add-on config..."
 
-    local validHostnameRegex="^(([a-zA-Z0-9]|[a-zA-Z0-9][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z0-9]|[A-Za-z0-9][A-Za-z0-9\-]*[A-Za-z0-9])$"
+    local validHostnameRegex="^(([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])\.)*([a-z0-9]|[a-z0-9][a-z0-9\-]*[a-z0-9])$"
```

Comment:
If the regex is changed, the error message after an unsuccessful check would probably have to be changed as well, right? Alternatively, it would also be possible to simply convert the string by using `bashio::string.lower`

Reply:
Agreed, I did include the message in my last commit. I also thought about simply changing it, but I would only do this if we are also able to actually change the config entry itself and not just write lowercase to Cloudflare. To do that is not super simple, especially for the additional_hosts array.
So I think it is easier and also clearer to just print the error message and have the users change it. Happy to hear your suggestion though